From 4c178782a779f013fafab476506f7d4dae372b8a Mon Sep 17 00:00:00 2001
From: Patrick Griffis <tingping@tingping.se>
Date: Sat, 17 Dec 2016 19:55:06 -0500
Subject: [PATCH] Don't combine compression with pixdata option for icon
 resources

This made minimal difference and is not recommended by upstream.
It also is affected by a regression in the latest gdk-pixbuf release.

https://bugzilla.gnome.org/show_bug.cgi?id=776105
---
 data/hexchat.gresource.xml | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/data/hexchat.gresource.xml b/data/hexchat.gresource.xml
index c125da2..5845da5 100644
--- a/data/hexchat.gresource.xml
+++ b/data/hexchat.gresource.xml
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gresources>
 	<gresource prefix="/icons">
-		<file alias="hexchat.png" preprocess="to-pixdata" compressed="true">icons/hexchat.png</file>
-		<file alias="book.png" preprocess="to-pixdata" compressed="true">icons/book.png</file>
+		<file alias="hexchat.png" preprocess="to-pixdata">icons/hexchat.png</file>
+		<file alias="book.png" preprocess="to-pixdata">icons/book.png</file>
 
-		<file alias="ulist_voice.png" preprocess="to-pixdata" compressed="true">icons/ulist_voice.png</file>
-		<file alias="ulist_halfop.png" preprocess="to-pixdata" compressed="true">icons/ulist_halfop.png</file>
-		<file alias="ulist_op.png" preprocess="to-pixdata" compressed="true">icons/ulist_op.png</file>
-		<file alias="ulist_owner.png" preprocess="to-pixdata" compressed="true">icons/ulist_owner.png</file>
-		<file alias="ulist_founder.png" preprocess="to-pixdata" compressed="true">icons/ulist_founder.png</file>
-		<file alias="ulist_netop.png" preprocess="to-pixdata" compressed="true">icons/ulist_netop.png</file>
+		<file alias="ulist_voice.png" preprocess="to-pixdata">icons/ulist_voice.png</file>
+		<file alias="ulist_halfop.png" preprocess="to-pixdata">icons/ulist_halfop.png</file>
+		<file alias="ulist_op.png" preprocess="to-pixdata">icons/ulist_op.png</file>
+		<file alias="ulist_owner.png" preprocess="to-pixdata">icons/ulist_owner.png</file>
+		<file alias="ulist_founder.png" preprocess="to-pixdata">icons/ulist_founder.png</file>
+		<file alias="ulist_netop.png" preprocess="to-pixdata">icons/ulist_netop.png</file>
 
-		<file alias="tray_fileoffer.png" preprocess="to-pixdata" compressed="true">icons/tray_fileoffer.png</file>
-		<file alias="tray_highlight.png" preprocess="to-pixdata" compressed="true">icons/tray_highlight.png</file>
-		<file alias="tray_message.png" preprocess="to-pixdata" compressed="true">icons/tray_message.png</file>
+		<file alias="tray_fileoffer.png" preprocess="to-pixdata">icons/tray_fileoffer.png</file>
+		<file alias="tray_highlight.png" preprocess="to-pixdata">icons/tray_highlight.png</file>
+		<file alias="tray_message.png" preprocess="to-pixdata">icons/tray_message.png</file>
 
 		<file alias="tree_channel.png" preprocess="to-pixdata">icons/tree_channel.png</file>
-		<file alias="tree_dialog.png" preprocess="to-pixdata" compressed="true">icons/tree_dialog.png</file>
-		<file alias="tree_server.png" preprocess="to-pixdata" compressed="true">icons/tree_server.png</file>
-		<file alias="tree_util.png" preprocess="to-pixdata" compressed="true">icons/tree_util.png</file>
+		<file alias="tree_dialog.png" preprocess="to-pixdata">icons/tree_dialog.png</file>
+		<file alias="tree_server.png" preprocess="to-pixdata">icons/tree_server.png</file>
+		<file alias="tree_util.png" preprocess="to-pixdata">icons/tree_util.png</file>
 	</gresource>
 </gresources>
